samples/flexibleEngine-Kafka.yml

```diff
@@ -61,6 +61,7 @@ flavors:
 # The contact details of the service.
 serviceProviderContactDetails:
   email: ["test@test.com"]
+eula:
```

Comment:
it need not be empty string always. It must work without the "eula" field completely in the OCL file. So we can remove files with these empty entries. Instead have couple of samples with a eula field and also text added to it.